### PR TITLE
Image based on BRT, documentation update and test suite

### PR DESCRIPTION
**Pull request by rpitonak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,55 +1,47 @@
-# DNS server
+Bind Docker image
```
Comment (TomasTomecek) [inline README.md]: can we `s/Docker/container/`?

File: tests/authoritative/sanity.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    :avocado: enable
+    """
+
+    def test1(self):
+        """
+        Simple sanity test
+        """
+
+        self.start()
+        self.run("ls / | grep bin")
```
Comment (TomasTomecek) [inline tests/authoritative/sanity.py]: why do we have this test?

File: tests/caching/requests.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        """
+        self.start()
+
+        time.sleep(5)
+
+        # cache request
+        self.run("dig @127.0.0.1 google.com")
+
+        # query time should be zero because of cache
+        self.run("dig @127.0.0.1 google.com | grep \"Query time: 0 msec\"")
```
Comment (TomasTomecek) [inline tests/caching/requests.py:47]: awesome, I love this! checking whether it's caching the actual value

**Review by TomasTomecek — CHANGES_REQUESTED** [review]
lovely work, just one nit and one question

**Comment by rpitonak** [comment]
Thank you for the review. Should be fixed now.

**Comment by TomasTomecek** [comment]
LGTM; shall I merge, anyone else wanna take a look?

**Review by TomasTomecek — APPROVED** [review]

**Pull request merged**